service/history/taskProcessor.go

```diff
@@ -116,7 +116,7 @@ func newTaskProcessor(
 		domainMetricsScopeCache: shard.GetService().GetDomainMetricsScopeCache(),
 		timeSource:              shard.GetTimeSource(),
 		workerNotificationChans: workerNotificationChans,
-		retryPolicy:             common.CreatePersistanceRetryPolicy(),
+		retryPolicy:             backoff.NewTwoPhaseRetryPolicy(),
```

Comment:
Do you plan to use this two phase retry policy for other components? This task processor will be deprecated soon as we switch to the priority task processor, which is using a very different retry policy for tasks. You can find the new retry policy in `common/util.go`

Reply:
For now, major created it for this processor to alleviate incident. No plan to put it to priority task processor before new processor goes alive.
It can be used in other components where retry forever happens, but would be better to have some metrics before switching to this policy.